admin-dev/themes/new-theme/js/pages/order/create/cart-editor.ts

```diff
@@ -30,6 +30,20 @@ import createOrderMap from './create-order-map';
 
 const {$} = window;
 
+export interface CartAddress {
```

Comment:
Maybe to better differentiate this class from the one in `admin-dev/themes/new-theme/js/pages/order/create/create-order-page.ts`
```suggestion
export interface CartAddressIds {
```